 .gitattributes

```diff
@@ -1 +0,0 @@
-CHANGELOG.md merge=union
```

Comment:
I'm actually moving this file - there was a space in the name 😮 (`mv ' .gitattributes' .gitattributes`) which was confusing the `git lfs` setup.